src/utils/tableDataUtils.ts

```diff
@@ -27,13 +27,17 @@ function getAPICallFunction({
     offset,
   }
 
-  if (mesh === 'all') {
+  if (!mesh || mesh === 'all') {
     return getAllEntities(params)
   } else if (getSingleEntity && query && query.length && mesh !== 'all') {
     return getSingleEntity(mesh, query, params)
   }
```

Comment:
```suggestion
  if (!mesh || mesh === 'all') {
    return getAllEntities(params)
  }

  if (getSingleEntity && query && query.length && mesh !== 'all') {
    return getSingleEntity(mesh, query, params)
  }
```
maybe like this, to be consistent, or if not, to add if statement below to another `else if`?

Reply:
yeah, you are right, especially that we return the value so there is no option that it will enter the second one after it pass first one.